app/pb_kits/playbook/pb_selectable_card/_selectable_card.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= content_tag(:div,
+  data: object.data,
+  class: object.classname) do %>
```

Comment:
Compare with other kits. I believe this is missing some things like ID, and possibly some other default props